packages/react-data-grid/src/CellAction.tsx

```diff
@@ -1,16 +1,23 @@
 import classNames from 'classnames';
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { ReactElement } from 'react';
 
-import CellActionShape from './common/prop-shapes/CellActionShape';
+interface Action {
+  icon: string | ReactElement;
+  actions: Array<{ text: string; callback: () => void }>;
```

Comment:
maybe add an interface for `{ text: string; callback: () => void }`?

Reply:
Done

Reply:
sure.